nsflow/run.py

```diff
@@ -48,8 +48,9 @@ def __init__(self):
         # Default Configuration
         self.config: Dict[str, Any] = {
             "server_host": os.getenv("NEURO_SAN_SERVER_HOST", "localhost"),
-            "server_port": int(os.getenv("NEURO_SAN_GRPC_SERVER_PORT", "30011")),
-            "server_connection": str(os.getenv("NEURO_SAN_SERVER_CONNECTION", "grpc")),
+            "server_grpc_port": int(os.getenv("NEURO_SAN_SERVER_GRPC_PORT", "30011")),
+            "server_http_port": int(os.getenv("NEURO_SAN_SERVER_HTTP_PORT", "8080")),
+            "server_connection": str(os.getenv("NEURO_SAN_SERVER_CONNECTION", "http")),
```

Comment:
Theoretically you shouldn't need both ports at the same time.
Consider keeping "server_port" with a different default value based on what the server connection is.